app/cdap/components/DirectiveInput/Components/AutoComplete/index.tsx

```diff
@@ -0,0 +1,236 @@
+/*
```

Comment:
i dont think we should name this component as auto complete
plz rename input panel or something

Reply:
Done